src/Uno.UWP/UI/Notifications/TileUpdateManager.Android.cs

```diff
@@ -0,0 +1,20 @@
+#if __ANDROID__
+
+namespace Windows.UI.Notifications
+{
+    public partial class TileUpdateManager
+    {
+        public static TileUpdater CreateTileUpdaterForApplication()
+        {
+            return new TileUpdater(null);
+        }
+
+        // for future implementation

```

Comment:
Is a todo needed here or a related task + link?

Reply:
I intended to extend work to secondary tiles, but as I cannot resolve problem with primary tile, I stop working on this PR :)
After making primary tile working, secondary tile implementation should be very easy (and is partly included in current code).